### PR TITLE
chore: Updates to security policy

### PR DESCRIPTION
**Pull request by binarylogic**
Signed-off-by: binarylogic <bjohnson@binarylogic.com>

### CODE REVIEW

File: SECURITY.md [hunk 1]
```diff
@@ -119,7 +128,23 @@ team members.
 All Vector team members are required to enable two-factor authentication
 for their Github accounts.
 
-## Development
+### Privilege Model
+
+Vector follows the [principle of least privilege][urls.least_privilege] for
+it's personnel access model. Vector maintains tiers user groups with granular
```
Comment (jamtur01) [inline SECURITY.md:133]: its

Comment (jamtur01) [inline SECURITY.md:133]: tiered

File: SECURITY.md [hunk 2]
```diff
@@ -119,7 +128,23 @@ team members.
...
+privileges to ensure users have access to the minimal resources necessary.
```
Comment (jamtur01) [inline SECURITY.md:134]: users only have access

File: SECURITY.md [hunk 3]
```diff
@@ -119,7 +128,23 @@ team members.
...
+
+### Third-Parties
+
+When used, third-parties must also adhere to this security policy. Access is
+based on the [principle of least privilege][urls.least_privilege] and removed
+when the contract ends.
+
+### Termination
+
+Upon termination,
```
Comment (jamtur01) [inline SECURITY.md]: Missing rest of sentence?

File: SECURITY.md [hunk 4]
```diff
@@ -168,45 +193,65 @@ Vector requires pull requests to pass all [automated checks](#automated-checks).
...
-##### Vulnerability Scans
+#### Vulnerability Scans & Security Advisories
 
-Vector implements an automated [`cargo deny` check][urls.cargo_deny]. This
-is part of the [Rust Security advisory database][urls.rust_sec].
+* Vector implements an automated [`cargo deny` check][urls.cargo_deny]. This
+  is part of the [Rust Security advisory database][urls.rust_sec].
+* Vector implements [Dependabot][urls.dependabot] which performs automated
+  upgrades on dependencies and [alerts][urls.dependabot_alerts] about any
+  dependency security vulernerabilities.
```
Comment (jamtur01) [inline SECURITY.md:201]: dependency-related

File: SECURITY.md [hunk 5]
```diff
@@ -168,45 +193,65 @@ Vector requires pull requests to pass all [automated checks](#automated-checks).
...
 
-Vector takes care to secure the build and release process to prevent unintended
-modifications.
+All builds run in an isolated sandbox that is destroyed after each use.
 
 ### Network Security
 
+#### Penetration Testing
+
+Vector performs qauterly pen tests on timber.io and vector.dev.
```
Comment (jamtur01) [inline SECURITY.md:224]: quarterly

File: docs/reference/components/transforms/split.cue [hunk 6]
```diff
@@ -3,6 +3,11 @@ package metadata
 components: transforms: split: {
 	title: "Split"
 
+	description:	"""
```
Comment (jamtur01) [inline docs/reference/components/transforms/split.cue]: Is this extraneous?

**Review by jamtur01 — CHANGES_REQUESTED** [review]
Some minor updates needed.

File: SECURITY.md (hunk 3 above)
Comment (binarylogic) [reply SECURITY.md]: Removed

File: docs/reference/components/transforms/split.cue (hunk 6 above)
Comment (binarylogic) [reply docs/reference/components/transforms/split.cue]: Removed

**Pull request merged**